functions/Get-DbaProductKey.ps1

```diff
@@ -1,7 +1,7 @@
 function Get-DbaProductKey {
     <#
     .SYNOPSIS
-        Gets SQL Server Product Keys from local or destination SQL Servers. Works with SQL Server 2005-2017
+        Gets SQL Server Product Keys from local or destination SQL Servers. Works with SQL Server 2008-2017
```

Comment:
Does this command not work on 2019 either?

Reply:
Good question!
I assume yes, but I will test it whenever I can.
If anyone have the opportunity to do it in the meantime please let us know.

Reply:
Confirmed. It works.
Changed to 2019.